stl/debugger/STL.natvis

```diff
@@ -1827,6 +1839,90 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
     </Expand>
   </Type>
 
+  <Type Name="std::default_sentinel_t">
+    <DisplayString>default sentinel</DisplayString>
+  </Type>
+
+  <Type Name="std::unreachable_sentinel_t">
+    <DisplayString>unreachable</DisplayString>
+  </Type>
+
+  <Type Name="std::ranges::empty_view&lt;*&gt;">
+    <DisplayString>{{ size=0 }}</DisplayString>
+  </Type>
+
+  <Type Name="std::ranges::single_view&lt;*&gt;">
+    <DisplayString>{{ size=1 }}</DisplayString>
+    <Expand>
+      <ArrayItems>
+        <Size>1</Size>
+        <ValuePointer>&amp;_Val._Val</ValuePointer>
+      </ArrayItems>
+    </Expand>
+  </Type>
+
+  <Type Name="std::ranges::iota_view&lt;*,*&gt;">
+    <Intrinsic Optional="true" Name="size" Expression="(size_t)(_Bound - _Value)" />
+    <DisplayString Optional="true">[{_Value}; {_Bound})</DisplayString>
+    <Expand>
+      <Item Optional="true" Name="[size]">size()</Item>
+      <CustomListItems Optional="true">
+        <Variable Name="val" InitialValue="_Value" />
+
+        <Size>size()</Size>
+        <Loop>
+          <Break Condition="val == _Bound" />
+          <Item>val</Item>
+          <Exec>++val</Exec>
+        </Loop>
+      </CustomListItems>
+    </Expand>
+  </Type>
+
+  <Type Name="std::ranges::iota_view&lt;*,std::unreachable_sentinel_t&gt;">
+    <DisplayString>[{_Value}; +&#8734;)</DisplayString>

```

Comment:
I have a mild preference to remove the `+` here - it seems inconsistent to me that `[5; +∞)` has a sign for infinity but not for `5`. (Ok to ignore this comment if you feel more strongly than I do.)

Reply:
I'm used to always having a sign before infinity, but I don't feel strongly about it.

Reply:
Fine, let's ship this. We can always change it later.